components/cache/index.php

```diff
@@ -23,10 +24,10 @@
     // which are loaded from the config class in the container.
     // More about the config class can be found in the config component; for now we will use an array
     $container['config'] = [
-        'cache.default' => 'file',
+        'cache.default'     => 'file',
         'cache.stores.file' => [
```

Comment:
I don't like this indentation style; I removed it from your previous commit intentionally. :)